gravityflow/gravityflowchecklists
# Put checklist+entry links through translation function

See HS#6115
Enables the link to new checklist form and entry to be opened in a new window, keeping the checklist view available. A refresh is still required on the checklist view to get an updated status.

To replicate:
- Create a checklist and add the shortcode [gravityflow page="checklists"]
- Click on individual form or submitted entry and note that the change the current page.

To test with branch in place:
- Modify .po file (or use Say What? plugin) to update the string
  **From:** `<a href="%1$s">%2$s</a>`
  **To:** `<a href="%1$s" target="_new">%2$s</a>`
- Repeat test and note that form and entry links open new window.
